tests/Double/Stub/Cache/LegacyCacheAnnotatedClass.php

```diff
@@ -77,23 +77,23 @@ public function cacheWithIdAndParameters(ParameterClassStub $param1, $param2): s
     }
 
     /**
-     * @Cache(handler="legacy_redis", namespace="'test-namespace'")
+     * @Cache(handler="legacy_array", namespace="'test-namespace'")
      */
     public function cacheWithNamespace(): string
     {
         return self::DATA;
     }
 
     /**
-     * @Cache(handler="legacy_redis", namespace="'test-namespace'", id="'toto'")
+     * @Cache(handler="legacy_array", namespace="'test-namespace'", id="'toto'")
      */
     public function cacheWithNamespaceAndId(): string
     {
         return self::DATA;
     }
 
     /**
-     * @Cache(handler="legacy_redis", namespace="'test-namespace' ~ param1.publicField")
+     * @Cache(handler="legacy_array", namespace="'test-namespace' ~ param1.publicField")
```

Comment:
we should maybe load the same handler with a different name to have a test for something other than `legacy_array`